vars/signArtifacts.groovy

```diff
@@ -30,8 +30,13 @@ void call(Map args = [:]) {
 }
 
 void importPGPKey(){
+    println("******** calling function to download key **********")
```

Comment:
Debugging statements?

Reply:
Yes, found them thanks to the test cases. Now removed.